kubehub/views/read_deploy_log_file.py

```diff
@@ -0,0 +1,26 @@
+from json import loads
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+
+from kubehub_backend.settings import K8S_DEPLOY_LOG_DIR
+from ..models.kubespray_deploy import KubesprayDeploy
+
+
+@csrf_exempt
+def read_deploy_log_file(request):
```

Comment:
get_deploy_logs more good name

Reply:
fixed in b8f5308 function name changed to get_deploy_logs